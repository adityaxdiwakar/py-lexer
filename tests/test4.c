int main() {
    int x=6;
    while (x >= 1) {
        x -=2;
    }
    printf(x);
    return 0;
}