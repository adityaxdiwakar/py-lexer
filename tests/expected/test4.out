INT_WORD
IDENT(main)
LPAREN
RPAREN
LBRACK
INT_WORD
IDENT(x)
EQUALS
INT(6)
SEMI
WHILE
LPAREN
IDENT(x)
GT
EQUALS
INT(1)
RPAREN
LBRACK
IDENT(x)
SUB
EQUALS
INT(2)
SEMI
RBRACK
IDENT(printf)
LPAREN
IDENT(x)
RPAREN
SEMI
RETURN
INT(0)
SEMI
RBRACK
